Blackbaud.AppFx.MajorGiving.Mobile.Sky/gruntfile.js

```diff
@@ -13,7 +13,7 @@ module.exports = function (grunt) {
         initialForceState = grunt.option("force"),
         testResultsFolder,
         isCrmInstallation = true,
-        vroot = '';
+        vroot = 'BBCRM_sp21_frogger/';
```

Comment:
Please back these changes out to leave the gruntfile in its default configuration.